engine/src/main/java/org/hibernate/validator/internal/xml/ConstrainedExecutableBuilder.java

```diff
@@ -55,7 +55,8 @@ private ConstrainedExecutableBuilder() {
 																			  String defaultPackage,
 																			  ParameterNameProvider parameterNameProvider,
 																			  ConstraintHelper constraintHelper,
-																			  AnnotationProcessingOptionsImpl annotationProcessingOptions) {
```

Comment:
The static nature of the XML-related classes required to change many places to pass on the class loader. It'd be a bit easier if these were instance methods...